portality/upgrade.py

```diff
@@ -55,10 +55,7 @@ def do_upgrade(definition, verbose):
             tasks = tdef.get("tasks", None)
             if tasks:
                 for func_call, kwargs in tasks.iteritems():
-                    if args:
```

Comment:
yup, it was something silly I did. It actually find args from the argparser below, so it does stuff so long as you have provided an argument...